ui/shared/Revision.jsx

```diff
@@ -94,13 +127,24 @@ export class Revision extends React.PureComponent {
         </span>
         <AuthorInitials title={`${name}: ${email}`} author={name} />
         <span
-          title={comment}
+          ref={this.spanRef}
           className={`ml-2 revision-comment overflow-hidden text-nowrap ${commentColor}`}
+          id={`revision${revision}`}
         >
           <em>
-            <BugLinkify>{comment}</BugLinkify>
+            <BugLinkify id={revision}>{comment}</BugLinkify>
           </em>
         </span>
+        {ready && (
+          <Tooltip
+            isOpen={tooltipOpen}
+            toggle={() => this.handleTooltip(comment)}
+            target={`revision${revision}`}
+          >
+            {/* {revisionComments} */}
```

Comment:
Here you would want to use `Object.entries` on `revisionComments.  And use `map` to iterate on each pair (`key` and `comment`).  The return value in the map function should return a `<div>` that looks like "Bug ### - Bugzilla comment here"

Reply:
Hm, revisionComments is intended to contain all the bugs in one RevisionList (I noticed I kept overwriting the state object when loading the bugs...oops). Mapping revisionComments will display all the bugs in the list as a tooltip, regardless if it belongs in that revision or not. Here's an illustration:
![image](https://user-images.githubusercontent.com/10367196/84603746-63065000-ae56-11ea-8815-66031c8e1836.png)

I'm going to instead loop over bugMatches, which holds the relevant bugs per revision. Let me know what you think of that.